spanner_orm/condition.py

```diff
@@ -225,13 +225,67 @@ def _validate(self, model):
     del model
 
 
+class OrCondition(Condition):
+  """Used to join multiple conditions with an OR in a Spanner query."""
+
+  def __init__(self, *condition_lists):
+    super().__init__()
+    if len(condition_lists) < 2:
+      raise error.SpannerError(
+          'OrCondition requires at least two lists of conditions')
+    self.condition_lists = condition_lists
+    self.all_conditions = []
+    for conditions in condition_lists:
+      self.all_conditions.extend(conditions)
+
+  def bind(self, model):
+    super().bind(model)
+    for condition in self.all_conditions:
+      condition.bind(model)
+
+  def _params(self):
+    result = {}
+    for condition in self.all_conditions:
+      condition.suffix = str(int(self.suffix or 0) + len(result))
```

Comment:
What is the suffix used for?  Unique names?

Reply:
yeah, because we use variable interpolation from spanner, we have to name each of the variables being interpolated. before we were just using the column name, but i fixed that recently to add these suffixes to make them all unique